Doc/reference/datamodel.rst

```diff
@@ -1145,7 +1145,7 @@ Basic customization
    class constructor expression.  If a base class has an :meth:`__init__`
    method, the derived class's :meth:`__init__` method, if any, must explicitly
    call it to ensure proper initialization of the base class part of the
-   instance; for example: ``BaseClass.__init__(self, [args...])``.
+   instance; for example: ``super().__init__(self, [args...])``.
```

Comment:
The switch to zero-arg super here means the explicit `self` is no longer needed in the call (whereas `__new__` is implicitly a static method, so it still needs the explicit `cls` arg)